integration-healthcheck/src/test/java/org/symphonyoss/integration/healthcheck/connectivity/PodConnectivityHealthIndicatorTest.java

```diff
@@ -30,28 +30,32 @@
 import org.symphonyoss.integration.model.yaml.IntegrationProperties;
 
 /**
- * Test class to validate {@link PodConnectivityVerifier}
+ * Test class to validate {@link PodConnectivityHealthIndicator}
  * Created by rsanchez on 23/11/16.
  */
 @RunWith(SpringRunner.class)
 @SpringBootTest
 @EnableConfigurationProperties
-@ContextConfiguration(classes = {IntegrationProperties.class, PodConnectivityVerifier.class})
-public class PodConnectivityVerifierTest {
+@ContextConfiguration(classes = {IntegrationProperties.class, PodConnectivityHealthIndicator.class})
+public class PodConnectivityHealthIndicatorTest {
 
   @MockBean
   private AuthenticationProxy authenticationProxy;
 
   @Autowired
-  private IntegrationProperties properties = new IntegrationProperties();
+  private IntegrationProperties properties;
```

Comment:
Same as above, it's probably not needed since it's being declared in @ContextConfiguration